doc/manual/src/command-ref/nix-env.md

```diff
@@ -140,11 +140,13 @@ have an effect.
     The command `nix-channel` places symlinks to the downloaded Nix
     expressions from each subscribed channel in this directory.
 
-  - `~/.nix-profile`\
+  - `$XDG_STATE_HOME/nix/profile`\
```

Comment:
This isn't true any more by default. Maybe keep this bit as-it-is, and document the redirections in the doc for the new option